src/admin/src/components/page/advisory/Advisory.js

```diff
@@ -477,6 +469,7 @@ export default function Advisory({
           setUrgencies([...urgencies]);
           const advisoryStatusData = res[10];
           const restrictedAdvisoryStatusCodes = ["INA", "APR"];
+          const desiredOrder = ['PUB', 'INA', 'DFT', 'APR', 'ARQ'];
           const tempAdvisoryStatuses = advisoryStatusData.map((s) => {
             let result = null;
```

Comment:
Setting this to and empty object instead of null seems to fix the error with submitters